README.md

```diff
@@ -37,12 +37,12 @@ in terms of performance. Here is a brief comparison:
 * __Addition.__
   All competing implementations of
   the modular addition involve divisions, while `mod` completely avoids
-  this costly operation. It makes difference even for small numbers;
-  e. g., `sum [1..10^7]` becomes 5x faster. For larger integers the speed up
+  this costly operation. This makes a difference even for small numbers;
+  e.g., `sum [1..10^7]` becomes 5x faster. For larger integers the speed up
```

Comment:
Please retain the space inside of e. g.

Reply:
Anything I could find online says that it's written "e.g.", do you still want me to revert this change?

Reply:
These are alternative forms and I have a personal preference. 
https://en.m.wiktionary.org/wiki/e._g.#English